src/ferc_xbrl_extractor/instance.py

```diff
@@ -390,16 +408,52 @@ def instances_from_zip(instance_path: Path | io.BytesIO) -> list[InstanceBuilder
 
     archive = zipfile.ZipFile(instance_path)
 
+    with archive.open("rssfeed") as f:
```

Comment:
Out of scope, but this is a place I think it would be really nice to have access to the `Datastore`. Given that we're basically requiring our specific archive layout for the input data, it would be nice to have a single unified interface to the archives.